### DISCUSSION TITLE
Add support for tags/topics

### DISCUSSION THREAD

**Discussion by ghalimi** [community/community#13097] [Discussions]
GitHub Discussions are awesome, and the Labels work great. Unfortunately, Labels are part of the metadata (schema), and work best when 20 or less are defined, mostly because they use colors, and it's virtually impossible to design a discrete color palette with more than 20 different colors.

If one wants to reproduce the user experience found on community websites like StackExchange, one needs proper tags/topics, which are part of the data (no schema), and can accommodate hundreds or even thousands of entries. These tags/topics should work like tags in StackExchange, whereby one can define a new tag/topic from a particular discussion.

The following repository emulates such tags/topics by listing them from a question's body, but this is clearly suboptimal:

https://github.com/stoic-doc/Community/discussions

And to be clear, such tags/topics should **not** replace Labels, they should complement them.

**Comment by marcastel** [comment]
[Agreed](https://github.com/orgs/community/discussions/3570#discussioncomment-1119032)... this remains the most wanted feature ... 2 years down the road
